packages/stencil/src/components/datacom-display-card/datacom-display-card-group.tsx

```diff
@@ -0,0 +1,22 @@
+import { Component, Host, h } from '@stencil/core';
+
+@Component({
+  tag: 'datacom-display-card-group',
+  styleUrl: 'datacom-display-card-group.css',
+  shadow: true,
+})
+export class DatacomDisplayCardGroup {
+  render() {
+    const Classes = {
+      'dc-display-card-group': true,
```

Comment:
If this is the only class being added AND it's always going to be applied, might as well just directly apply the class on line 15. You only really want to do this approach if you've got multiple conditional classes to render.